objcgen/driver.cs

```diff
@@ -521,6 +521,7 @@ public int Compile ()
 						mtouch.Append ("--msym:false ");
 						mtouch.Append ("--nosymbolstrip ");
 						mtouch.Append ($"--embeddinator ");
+						mtouch.Append ($"--xml={Path.Combine (OutputDirectory, "linker.xml")} ");
```

Comment:
Yep I should have noted this as a workaround since I guess we also need to accept a `linker.xml` from the user and merge our definitions. This was more to get my testing on iOS going 😃